autoload/lsp/ui/vim/completion.vim

```diff
@@ -0,0 +1,240 @@
+" vint: -ProhibitUnusedVariable
+"
+let s:context = {}
+
+function! lsp#ui#vim#completion#setup() abort
+  augroup lsp_ui_vim_completion
+    autocmd!
+    autocmd CompleteDone * call s:on_complete_done()
+  augroup END
+endfunction
+
+"
+" After CompleteDone, v:complete_item's word has been inserted into the line.
+" Yet not inserted commit characters.
+"
+" below example uses # as cursor position.
```

Comment:
It would be good to use `|` instead of `#` for cursor position since most of the cursors actually looks like `|`.